lld/COFF/Writer.cpp

```diff
@@ -2584,6 +2586,16 @@ void Writer::createDynamicRelocs() {
                          coffHeaderOffset + offsetof(coff_file_header, Machine),
                          AMD64);
 
+  if (ctx.symtab.entry || ctx.hybridSymtab->entry) {
+    Defined *entrySym = ctx.hybridSymtab->entry
+                            ? dyn_cast<Defined>(ctx.hybridSymtab->entry)
+                            : nullptr;
```

Comment:
Wouldn't a `dyn_cast_or_null<Defined>(ctx.hybridSymtab->entry)` be equivalent to this whole ternary expression?

Reply:
Oh, right, it could even be `cast_or_null`.